eth/main.cpp

```diff
@@ -1108,10 +1108,11 @@ int main(int argc, char** argv)
 			}
 		}
 
-		while (web3.ethereum()->blockQueue().items().first + web3.ethereum()->blockQueue().items().second > 0)
```

Comment:
This check was remaining true forever in case when 1 block is added to BlockQueue, is valid but with unknown parent.